routes/web.php

```diff
@@ -22,6 +22,8 @@
 Route::get('/phpinfo', 'IndexController@phpinfo')->middleware('admin');
 Route::get('/gallery', 'IndexController@gallery')->name('gallery');
 
+Route::get('/images/{img}', 'ImageController@view');
```

Comment:
Sorry, I may have asked this before, but what is this for? It doesn't seem to be defined anywhere...

Reply:
Yeah that was the beginning of the auth based system before I considered symlinks. I will remove it.